examples/gallery/symbols/text_symbols.py

```diff
@@ -10,10 +10,10 @@
 different characters) and *string* can be a letter or a text string
 (less than 256 characters). Optionally, you can append **+f**\ *font* to
 select a particular font [Default is :gmt-term:`FONT_ANNOT_PRIMARY`] as well as
-**+j**\ *justify* to change the justification [Default is CM]. Outline
-and fill color of the text symbols can be customized via the ``pen``
-and ``color`` parameters, respectively.
-
+**+j**\ *justify* to change the justification [Default is CM]. Outline color of
+the text symbols can also be defined via the ``style`` parameter [Default is
```

Comment:
```suggestion
the text symbols can be defined via the ``style`` parameter [Default is
```
Is there another way that the outline color can be set? If not, this shouldn't say "also"

Reply:
I'm not fully sure, the GMT documentations is not really detailed at this point.